src/shared/product.util.js

```diff
@@ -0,0 +1,39 @@
+export const getSortedProducts = (products = [], priceSortCriteria) => {
+  const sortedList = [...products];
+  if (priceSortCriteria && priceSortCriteria === 'lowToHighPrice') {
+    sortedList.sort((a, b) => a.price - b.price);
```

Comment:
`a` and `b` variables could be named a bit better as `productOne` or `firstProduct` and `productTwo` or `secondProduct`